### PR TITLE
Fixes possible NullPointer in ProtoType.java

### PR DESCRIPTION
**Pull request by slebs**

### CODE REVIEW

File: src/main/java/org/ebml/ProtoType.java [hunk 1]
```diff
@@ -49,6 +49,11 @@ public static Element getInstance(final ByteBuffer type)
     final long codename = EBMLReader.parseEBMLCode(type);
     final ProtoType<? extends Element> eType = CLASS_MAP.get(Long.valueOf(codename));
     LOG.trace("Got codename {}, for element type {}", codename, eType.name);
+
+    if (eType == null) {
```
Comment (mbunkus) [inline src/main/java/org/ebml/ProtoType.java:52]: Thanks for the PR.

I don't know (nor use) the Java code, but just from looking at the surrounding lines it seems clear to me that the check has to be moved upwards, at least before the `LOG.trace…` line as that one dereferences `eType` already.

**Review by mbunkus — CHANGES_REQUESTED** [review]

File: src/main/java/org/ebml/ProtoType.java (hunk 1 above)
Comment (slebs) [reply src/main/java/org/ebml/ProtoType.java:52]: I moved the Nullpointer check onto the appropriate position in the code. To hide my sloppiness i amended the commit. Would u consider merging it now?

File: src/main/java/org/ebml/ProtoType.java [hunk 2]
```diff
@@ -48,6 +48,11 @@ public static Element getInstance(final ByteBuffer type)
   {
     final long codename = EBMLReader.parseEBMLCode(type);
     final ProtoType<? extends Element> eType = CLASS_MAP.get(Long.valueOf(codename));
+
+    if (eType == null) {
+      return null;
```
Comment (SiWoC) [inline src/main/java/org/ebml/ProtoType.java:53]: I would also suggest returning Void to have the rest of the file getting parsed correctly and logging the unknown codename
```suggestion
      LOG.error("Unknown codename [" + codename + "]=[" + EBMLReader.bytesToHex(type.array()) + "], returning [Void]");
      return MatroskaDocTypes.Void.getInstance();
    
```

**Comment by mbunkus** [comment]
Thanks.

**Pull request merged**